plugin/util.go

```diff
@@ -68,6 +88,33 @@ func toLink(args Args) string {
 	return args.Commit.Link
 }
 
+// helper function ExtractInstanceName extracts the instance name from the provided URL
+// or returns the instance name directly
+func ExtractInstanceName(instance string) string {
+	// Check if the instance is a full URL
+	if strings.Contains(instance, "://") {
+		parsedURL, err := url.Parse(instance)
+		if err == nil {
+			// Return the host part without the top-level domain
+			hostParts := strings.Split(parsedURL.Hostname(), ".")
+			if len(hostParts) > 0 {
+				return hostParts[0] // Return the first part as the instance name
+			}
+		} else {
+			// Log the error if URL parsing fails
+			log.Printf("Error parsing URL '%s': %v", instance, err)
```

Comment:
consider using the logger like logrus to maintain consistency across the plugin

Reply:
done